README.md

```diff
@@ -1,3 +1,46 @@
+
 # Dat Enhancement Proposals
 
-Dat Enhancement Proposals. Contains all specs for the Dat protocol, including drafts.
+[![#dat IRC channel on freenode](https://img.shields.io/badge/irc%20channel-%23dat%20on%20freenode-blue.svg)](http://webchat.freenode.net/?channels=dat)
+[![datproject/discussions](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/datproject/discussions?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![docs](https://img.shields.io/badge/Dat%20Project-Docs-green.svg)](http://docs.dat-data.com)
+
+This repository contains a series of "Dat Enhancement Proposal" (DEP)
+documents, part of the dat protocol development and standardization process.
+
+Accepted proposals will be viewable at [datprotocol.com](https://datprotocol.com).
+New (draft) proposals and discussion can be viewed on [github][github-deps]
+under "issues" and "pull-requests".
```

Comment:
All DEP proposal discussion should happen in PRs so we don't get discussion fractured.